### PR TITLE
[MO] - [system test] -> additional check for connectTimeout and readT…

### PR DESCRIPTION
**Pull request by see-quick**
…imeout

Signed-off-by: morsak <xorsak02@stud.fit.vutbr.cz>

### Type of change

- Enhancement / new feature

### Description

This PR adds checks for new options in [`0.10.0` ](https://github.com/strimzi/strimzi-kafka-oauth/releases/tag/0.10.0). Specifically, connect and read timeout. The overall functionality is tested in the [strimzi-oauth-repo](https://github.com/strimzi/strimzi-kafka-oauth) but here I adding a check that each option is properly propagated and detected in the components (i.e., `KafkaBridge`, `KafkaConnect`, `KafkaMirrorMaker`, `KafkaMirrorMaker2`..). This check is verified inside the logs of each component.

### Checklist

- [x] Make sure all tests pass

### CODE REVIEW

File: systemtest/src/test/java/io/strimzi/systemtest/security/oauth/OauthPlainIsolatedST.java [hunk 1]
```diff
@@ -302,18 +308,26 @@ void testProducerConsumerConnect(ExtensionContext extensionContext) {
                         .withSecretName(CONNECT_OAUTH_SECRET)
                         .withKey(OAUTH_KEY)
                     .endClientSecret()
+                    .withConnectTimeoutSeconds(CONNECT_TIMEOUT_S)
+                    .withReadTimeoutSeconds(READ_TIMEOUT_S)
                 .endKafkaClientAuthenticationOAuth()
                 .withTls(null)
+                .withNewInlineLogging()
+                    .addToLoggers("connect.root.logger.level", "DEBUG")
```
Comment (Frawless) [inline systemtest/src/test/java/io/strimzi/systemtest/security/oauth/OauthPlainIsolatedST.java:316]: I think you should add comment why this is needed

File: systemtest/src/test/java/io/strimzi/systemtest/security/oauth/OauthPlainIsolatedST.java [hunk 2]
```diff
@@ -629,6 +659,10 @@ void testProducerConsumerBridge(ExtensionContext extensionContext) {
         JobUtils.deleteJobWithWait(INFRA_NAMESPACE, consumerName);
 
         resourceManager.createResource(extensionContext, KafkaClientsTemplates.kafkaClients(INFRA_NAMESPACE, false, kafkaClientsName).build());
+
+        InlineLogging ilDebug = new InlineLogging();
+        ilDebug.setLoggers(Map.of("rootLogger.level", "DEBUG"));
```
Comment (Frawless) [inline systemtest/src/test/java/io/strimzi/systemtest/security/oauth/OauthPlainIsolatedST.java:664]: Same as above

**Review by Frawless — APPROVED** [review]

**Review by mstruk — APPROVED** [review]
Didn't try to run it, but looks good.

**Pull request merged**